tracing/custom-tracing-subscriber/src/main.rs

```diff
@@ -0,0 +1,32 @@
+use actix_web::{get, web::ServiceConfig};
+use shuttle_actix_web::ShuttleActixWeb;
+use tracing_subscriber::filter::{EnvFilter, LevelFilter};
+
+#[tracing::instrument(name = "hello")]
+#[get("/")]
+async fn hello_world() -> &'static str {
+    let msg = "Hello World!";
+    tracing::info!("{msg}");
+    msg
+}
+
+#[shuttle_runtime::main]
+async fn actix_web() -> ShuttleActixWeb<impl FnOnce(&mut ServiceConfig) + Send + Clone + 'static> {
+    // Normally we write to stdout for Shuttle to record the logs (i.e. println!)
+    // But we can use a fmt subscriber instead which has sane defaults for applications and also
+    // writes to stdout.
```

Comment:
What do you think about this? I think we can keep it somewhat simple, and we will explain more in depth in our docs page, where we'll also link to this example.
```suggestion
    // We need to write to stdout for Shuttle to record our logs, so we use the 
    // tracing::fmt subscriber which has sane defaults for applications and also
    // writes to stdout.
```

Reply:
6a516838dce5927185d95971109885698e0c1b26